typescript/simple-mercari-web/src/components/ItemList.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+import { Item, fetchItems } from '~/api';
+
+const PLACEHOLDER_IMAGE = import.meta.env.VITE_FRONTEND_URL + '/logo192.png';
+
+interface Prop {
+  reload: boolean;
+  onLoadCompleted: () => void;
+}
+
+export const ItemList: React.FC<Prop> = (props) => {
+  const { reload, onLoadCompleted } = props;
```

Comment:
Personal preference: Since `React.FC` is no longer needed. We can define the props explicitly

```suggestion
interface Props {
  reload: boolean;
  onLoadCompleted: () => void;
}

export const ItemList = ({reload, onLoadCompleted}: Props) => {
```

Reply:
God !!! Fixed it!! Thank you~~~~